### PR TITLE
Added NOMAD Pool.json

### PR DESCRIPTION
**Pull request by nomadpool**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -4207,5 +4207,19 @@
...
+   "ticker": "NOMAD",
+   "poolId": "bbebbbf81c42de5b84fbbc82c4feab78f8bd8bcf8b5af7c73a06664a",
+   "social": {
+     "website": "https://www.nomadpool.io/",
+     "twitter": "zhekree",
+     "facebook": "",
+     "instagram": "",
+     "youtube": "zhekson"
+   },
+   "telegram": "",
```
Comment (Bez625) [inline registry.json:4221]: As per the alliance requirements you must provide a valid telegram id. This must be for an individual and not a group or channel

File: registry.json [hunk 2]
```diff
@@ -4207,5 +4207,19 @@
...
+   "operatorName": "zhekson",
```
Comment (Bez625) [inline registry.json:4222]: The operator name should be a real full name, first name or leave blank.

**Review by Bez625 — CHANGES_REQUESTED** [review]

File: registry.json [hunk 3]
```diff
@@ -4207,5 +4207,19 @@
    "telegram": "xbglowx",
    "operatorName": "Brian",
    "github": "xbglowx"
+  },
+  {
+   "ticker": "NOMAD",
+   "poolId": "bbebbbf81c42de5b84fbbc82c4feab78f8bd8bcf8b5af7c73a06664a",
+   "social": {
+     "website": "https://www.nomadpool.io/",
```
Comment (Bez625) [inline registry.json:4215]: Hey I'm getting a 404 on this website, i'll approve once you confirm it's all good. Mostly doing this to make you aware

**Review by Bez625 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 3 above)
Comment (nomadpool) [reply registry.json:4215]: Thanks for the heads up! I am aware of this - website is currently under construction, will be ready soon. Got the domain name ready beforehand though.

**Pull request merged**